docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+
+**S gate**
+
+The S gate applies a phase of $i$ to the $\left|1\right\rangle$ state. It is equivalent to RZ for the angle $\pi/2$. Note that S=P($\pi/2$).
+
+| Composer reference                                | OpenQASM reference | Q-sphere | Note about q-sphere representations                                                                                                                                                                                                     |
+| ------------------------------------------------- | ------------------ | ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![s\_gate](/docs/images/guides/composer/c3217084-41c0-4821-80fc-b0852a8c2cd4.avif) | `s q[0];`          | ![image25](/docs/images/guides/composer/ce2da6b1-c21b-4ec0-be82-984a874f02af.avif)                              | The q-sphere representation shows the state after the gate operates on the initial equal superposition state $\frac{1}{\sqrt{2^{n}}}\sum_{i=0}^{2^{n}-1}\vert i\rangle, $ where $n$ is the number of qubits needed to support the gate. |
+
+**Z gate**
+
+The Pauli Z gate acts as identity on the $\left|0\right\rangle$ state and multiplies the sign of the $\left|1\right\rangle$ state by -1. It therefore flips the $\left|+\right\rangle$ and $\left|-\right\rangle$ states. In the +/- basis, it plays the same role as the NOT gate in the $\left|0\right\rangle$/$\left|1\right\rangle$ basis.
+
+| Composer reference                                | OpenQASM reference | Q-sphere | Note about q-sphere representations                                                                                                                                                                                                     |
+| ------------------------------------------------- | ------------------ | ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![z\_gate](/docs/images/guides/composer/941c8820-65bc-444c-890b-b445b9958085.avif) | `z q[0];`          | ![image23](/docs/images/guides/composer/84b92ecb-3b62-4cb3-9c01-549866681816.avif)                              | The q-sphere representation shows the state after the gate operates on the initial equal superposition state $\frac{1}{\sqrt{2^{n}}}\sum_{i=0}^{2^{n}-1}\vert i\rangle, $ where $n$ is the number of qubits needed to support the gate. |
+
+**$T^{\dagger}$ gate**
+
+Also known as the Tdg or T-dagger gate.
+
+The inverse of the T gate.
```

Comment:
These sentences confuse me a little because they are not complete sentences. Should they use `:`?

Reply:
I think it's ok - they are typical fragments of a reference/glossary-type nature. But if we get more confusion about them in the future we'll rewrite.